$OpenBSD: patch-tests_Makefile,v 1.5 2009/06/03 22:43:59 jasper Exp $

- Remove GNU make crap that never applied to us anyway.

--- tests/Makefile.orig	Wed Jun  3 16:41:44 2009
+++ tests/Makefile	Wed Jun  3 16:42:50 2009
@@ -6,38 +6,20 @@ PROGS = test1 test2 test3 test4 test5 test6 test7 test
         test18 test19 test20 test21 test22 test23 test24 test25 \
         test26 test27 test28 test29 test30 test31 test32 test33 \
         test34 test35 test36 test37 test38 test39
-CFLAGS = -I$(HASHDIR) 
-CFLAGS += -O3
-#CFLAGS += -g
-CFLAGS += -Wall 
+CFLAGS += -I$(HASHDIR) 
+CFLAGS += -O2
+CFLAGS += -Wall
 CFLAGS += ${EXTRA_CFLAGS}
 
-ifeq ($(HASH_DEBUG),1)
-CFLAGS += -DHASH_DEBUG=1
-endif
-
-ifeq ($(HASH_PEDANTIC),1)
-CFLAGS += -pedantic 
-endif
-
 TEST_TARGET=run_tests
 TESTS=./do_tests
 
-# Prefer 64-bit compilation on Mac OS X (not necessary, just faster)
-ifneq ($(strip $(shell $(CC) -v 2>&1 |egrep "i[0-9]+-apple-darwin")),)
-  CFLAGS += -m64
-endif
-
-# detect Cygwin
-ifneq ($(strip $(shell $(CC) -v 2>&1 |grep "cygwin")),)
-  TESTS=./do_tests.cygwin
-endif
-
+# gnu make idiom...
 # detect MinGW
-ifneq ($(strip $(shell $(CC) -v 2>&1 |grep "mingw")),)
-  TEST_TARGET=run_tests_mingw
-  TESTS=./do_tests.mingw
-endif
+#ifneq ($(strip $(shell $(CC) -v 2>&1 |grep "mingw")),)
+#  TEST_TARGET=run_tests_mingw
+#  TESTS=./do_tests.mingw
+#endif
 
 
 
